resources/views/setup/user.blade.php

```diff
@@ -25,6 +26,68 @@
       </div>
     </div>
 
+  <div class="row">
+
+    <!-- Language -->
+    <div class="form-group col-lg-6 {{$errors->has('default_language') ? 'error' : ''}}">
+      {{ Form::label('site_name', trans('admin/settings/general.default_language')) }}
```

Comment:
Also, I think the label name is wrong here?

`{{ Form::label('locale', trans('admin/settings/general.default_language')) }}`

should be

`{{ Form::label('site_name', trans('admin/settings/general.default_language')) }}`